### ISSUE TITLE
Why doesn't the loss function go down？

### ISSUE THREAD

**Issue by LYJvincent** [Tramac/Fast-SCNN-pytorch#14]
Hi！Excuse me！
I use this network to detect defects.Use MixSoftmaxCrossEntropyLoss loss function, training to 400 eopch, but loss function has been drop down between 0.5 to 0.7.May I ask what the problem is?

**Comment by LYJvincent** [comment]
MixSoftmaxCrossEntropyOHEMLoss and MixSoftmaxCrossEntropyLoss have the same effect, The learning rate is not going down

**Comment by feitiandemiaomi** [comment]
@LYJvincent I meet the same problem with you, I use my dataset to train it , the loss  always shakes.
![loss](https://user-images.githubusercontent.com/18325462/63233474-e705a080-c262-11e9-9cc5-d8355182d757.png)

**Comment by Tramac** [comment]
How much is the learning rate?

**Comment by feitiandemiaomi** [comment]
@Tramac lr is 0.01 ,the effect of detect is ok ,though it shakes   
![img_save](https://user-images.githubusercontent.com/18325462/63234516-1ec31700-c268-11e9-90f6-2e059d5184da.jpg)
![example 00004](https://user-images.githubusercontent.com/18325462/63234518-21257100-c268-11e9-876b-4418b45c1265.jpg)

**Comment by feitiandemiaomi** [comment]
@Tramac  I'm wrong ,the lr=0.001 and I use the adam optimizer

**Comment by LYJvincent** [comment]
@Tramac  the lr = 1e-4, now I get better results。thanks！

**Comment by feitiandemiaomi** [comment]
@LYJvincent What the values of  miou can achieve?

**Comment by LYJvincent** [comment]
@feitiandemiaomi  current mean_iou is 0.97602%， about 3.5 hour

**Comment by LYJvincent** [comment]
@feitiandemiaomi  is just for my database not cityspaces

**Comment by Tramac** [comment]
> @feitiandemiaomi current mean_iou is 0.97602%， about 3.5 hour

0.97602% or 97.602%?

**Comment by LYJvincent** [comment]
@Tramac Oh,sorry, is my fault。not 0.97602%, but 97.602%

**Comment by feitiandemiaomi** [comment]
@LYJvincent Can you share hyperparameters ?

**Comment by feitiandemiaomi** [comment]
@LYJvincent The  paper's miou is 69%, your result is  amazing. Hope to learn it ,Thanks

**Comment by Tramac** [comment]
It depends on the datasets.

**Comment by CuttlefishXuan** [comment]
@feitiandemiaomi Hi, I meet the same problem as you, the training loss fluctuates between 0.9 and 1.1
Have you solved this problem? Could you get the better result

**Comment by Alohomora0130** [comment]
> @feitiandemiaomi Hi, I meet the same problem as you. Have you solve this problem? If you have solve this problem, can you share me with your learning rate and batch_size? Thanks!

**Comment by Alohomora0130** [comment]
@CuttlefishXuan Hi! Have you solve this problem? Can you tell me your learning rate and batch_size?T hank you very much!
